modules/seqexec/server/src/main/scala/seqexec/server/SequenceConfiguration.scala

```diff
@@ -51,7 +51,10 @@ trait SequenceConfiguration {
     }.getOrElse(StepState.Failed("Logical error reading step status"))
 
   def extractWavelength(config: CleanConfig): Option[Wavelength] =
-    config.extractAs[Wavelength](OBSERVING_WAVELENGTH_KEY).toOption
+    config
+      .extractAs[String](OBSERVING_WAVELENGTH_KEY)
+      .flatMap(v => Either.catchNonFatal(v.toDouble))
+      .map(Wavelength.fromMicrons[Double](_)).toOption
```

Comment:
Just to know what's happening here: what happens if parsing fails?

Reply:
The response gets to `None` which is what you want. but now that you mention it maybe we should log something